docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/attribution/index.md

```diff
@@ -58,6 +58,14 @@ vars:
     snowplow__conversion_hosts: ['mysite.com']
     snowplow__path_transforms: {'exposure_path' : null}
 ```
+<details>
+    <summary>Decide on running optional reporting models</summary>
+
+By default the package creates a View used for BI reporting called `snowplow__enable_attribution_overview`. In case you would like to use the Attribution Modeling Data App, this is not required to be enabled as app will take care of querying this data for you directly.
+
+There is also a `snowplow__enable_paths_to_non_conversion` variable to produce a drop and recompute table for more advanced analysis which is disabled by default. The data app does not yet support recreating the same data, therefore, it needs to be enabled by setting the default variable to `true`.
+</details>
```

Comment:
Can we maybe keep this commented out until we have the actual data app itself in the docs, otherwise this may confuse people

Reply:
On Matus's example let me present you the PR for this: https://github.com/snowplow/documentation/pull/817